src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/WT/AltimeterIndicator.js

```diff
@@ -1551,39 +1600,59 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
+                    this.pressureSVGUnits.removeAttribute("stroke-width", "5px");
                     this._baroPresetChanged = baroPreset;
                 }
                 if (units == "millibar") {
-                    this.pressureSVG.textContent = "1013";
+                    this.pressureSVGLeftPart.textContent = "1013";
+                    this.pressureSVGCenterPart.textContent = "";
+                    this.pressureSVGRightPart.textContent = "";
                     this.pressureSVGUnits.textContent = "STD";
-                    this.pressurePreset.textContent = baroPreset.toFixed(0);
+                    this.pressurePresetLeftPart.textContent = baroPreset.toFixed(0);
+                    this.pressurePresetCenterPart.textContent = "";
+                    this.pressurePresetRightPart.textContent = "";
                     this.pressurePresetUnits.textContent = "HPA";
                 }
                 else {
                     let rounded = Math.round((baroPreset / 33.8639) * 1000) / 1000; //Converts baroPreset to inHg then rounds up to hundreths so it matches what the sim will show for pressure.
-                    this.pressureSVG.textContent = "29.92";
+                    this.pressureSVGLeftPart.textContent = "29";
+                    this.pressureSVGCenterPart.textContent = ".";
+                    this.pressureSVGRightPart.textContent = "92";
                     this.pressureSVGUnits.textContent = "STD";
-                    this.pressurePreset.textContent = rounded.toFixed(2);
+                    let pressureSetting = (rounded.toFixed(2).toString());
+                    let leftPart = pressureSetting.slice(0,2);
+                    let rightPart = pressureSetting.slice(3,5);
+                    this.pressurePresetLeftPart.textContent = leftPart;
+                    this.pressurePresetCenterPart.textContent = ".";
+                    this.pressurePresetRightPart.textContent = rightPart;
+
                     this.pressurePresetUnits.textContent = "IN";
                 }
             }
             else {
                 this.pressureBoxGroup.setAttribute("visibility", "hidden");
-                this.pressureSVG.setAttribute("stroke", "black");
-                this.pressureSVG.setAttribute("stroke-width", "7px");
+                this.pressureSVGLeftPart.setAttribute("stroke", "black");
+                this.pressureSVGLeftPart.setAttribute("stroke-width", "5px");
+                this.pressureSVGCenterPart.setAttribute("stroke", "black");
+                this.pressureSVGCenterPart.setAttribute("stroke-width", "4px");
+                this.pressureSVGRightPart.setAttribute("stroke", "black");
+                this.pressureSVGRightPart.setAttribute("stroke-width", "5px");
                 this.pressureSVGUnits.setAttribute("stroke", "black");
-                this.pressureSVGUnits.setAttribute("stroke-width", "7px");
+                this.pressureSVGUnits.setAttribute("stroke-width", "5px");

```

Comment:
These attribute lines can be removed.

Reply:
I think they are removed when the preset box is opened and then added back when closed. Since the stroke is black it probably would not show in the preset box anyway Should I remove this code and test what happens?

Reply:
Yes, exactly. I have removed them on my end and didn't see any negative effect. You can verify that and commit.

Reply:
Lines removed.